integration-tests/jpa-postgresql-vault/src/test/java/io/quarkus/it/jpa/postgresql/VaultTestResource.java

```diff
@@ -0,0 +1,29 @@
+package io.quarkus.it.jpa.postgresql;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import io.quarkus.test.common.QuarkusTestResourceLifecycleManager;
+import io.quarkus.vault.VaultTestExtension;
+
+public class VaultTestResource implements QuarkusTestResourceLifecycleManager {
+
+    public static final String GRAALVM_JRE_LIB_AMD_64 = "/opt/graalvm/jre/lib/amd64";
```

Comment:
How do we expect this to be portable?

Reply:
by default tests are disabled on windows as discussed in https://github.com/quarkusio/quarkus/pull/2897#discussion_r326310602, and tls is activated only if system property vault-test-extension.use-tls is set to true (false by default in integration-tests/vault)

see 
- https://github.com/vsevel/quarkus/blob/master/integration-tests/vault/pom.xml#L127
- https://github.com/vsevel/quarkus/blob/master/test-framework/vault/src/main/java/io/quarkus/vault/VaultTestExtension.java#L238-L240

and issues https://github.com/quarkusio/quarkus/issues/3797 and https://github.com/quarkusio/quarkus/issues/3796